packages/database/src/migrations/20201201082830-AddProjectPalauOlangch-modifies-data.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+import { codeToId, insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const COUNTRY_CODE = 'PW';
+const PROJECT_CODE = 'olangch_palau';
+const PROJECT_NAME = 'MoH Project Olangch';
+const DASHBOARDGROUP_NAME = 'General';
+const DASHBOARDGROUP_CODE = 'PW_Olangch';
+const DASHBOARDGROUP_COUNTRY_CODE = 'PW_Olangch_Country';
+const USER_GROUP = 'Donor';
+
+export const hierarchyNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM entity_hierarchy WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+exports.up = async function (db) {
+  await insertObject(db, 'dashboardGroup', {
+    organisationLevel: 'Project',
+    userGroup: USER_GROUP,
+    organisationUnitCode: PROJECT_CODE,
+    dashboardReports: '{project_details}',
+    name: DASHBOARDGROUP_NAME,
+    code: DASHBOARDGROUP_CODE,
+    projectCodes: `{${PROJECT_CODE}}`,
+  });
+  await insertObject(db, 'dashboardGroup', {
+    organisationLevel: 'Country',
+    userGroup: USER_GROUP,
+    organisationUnitCode: COUNTRY_CODE,
+    dashboardReports: '{23,19,8,26}',
+    name: DASHBOARDGROUP_NAME,
+    code: DASHBOARDGROUP_COUNTRY_CODE,
+    projectCodes: `{${PROJECT_CODE}}`,
+  });
+  await insertObject(db, 'entity', {
+    id: generateId(),
+    code: PROJECT_CODE,
+    parent_id: await codeToId(db, 'entity', 'World'),
+    name: PROJECT_NAME,
+    type: 'project',
+  });
+
+  await insertObject(db, 'entity_hierarchy', {
+    id: generateId(),
+    name: PROJECT_CODE,
+  });
+  await insertObject(db, 'entity_relation', {
+    id: generateId(),
+    parent_id: await codeToId(db, 'entity', PROJECT_CODE),
+    child_id: await codeToId(db, 'entity', COUNTRY_CODE),
+    entity_hierarchy_id: await hierarchyNameToId(db, PROJECT_CODE),
+  });
+  await insertObject(db, 'project', {
+    id: generateId(),
+    code: PROJECT_CODE,
+    description: 'HIS for Palau’s public health data, integrating DHIS2 and mSupply.',
+    sort_order: 10,
+    image_url: '', // 'https://tupaia.s3-ap-southeast-2.amazonaws.com/uploads/palau_olangch_background.png',
```

Comment:
Logo image is a temp image.
Background image hasn't been supplied yet. Another option would be to add a blank image with this name to aws - so that when real one comes through, doesn't need code change.

Reply:
Went with uploading a blank image.